HISTORY.md

```diff
@@ -0,0 +1,60 @@
+# Change log
```

Comment:
You don't want this header and `All notable changes to this project` text in the HISTORY.md file or it will appear twice in the generated CHANGELOG.md